tests/integration/test_detailed_records.py

```diff
@@ -37,8 +37,8 @@ def test_all_records_were_loaded(app):
 
 
 def test_all_records_are_valid(app):
-    invalid = InspireProdRecords.query.filter(InspireProdRecords.valid == False)
-    recids = [el.recid for el in invalid]
+    invalid = InspireProdRecords.query.filter(InspireProdRecords.valid == False).values(InspireProdRecords.recid)
+    recids = [el[0] for el in invalid]
```

Comment:
Speed :smile: (it's 30 times faster). I know it's useless (1ms instead of 30ms) but since I already took the pain to write it because I didn't know that test was already there... I can remove it from the commit if you prefer.